sharding-core/src/main/java/in/cleartax/dropwizard/sharding/transactions/TransactionRunner.java

```diff
@@ -23,16 +23,18 @@
 import in.cleartax.dropwizard.sharding.hibernate.MultiTenantUnitOfWorkAwareProxyFactory;
 import io.dropwizard.hibernate.UnitOfWork;
 import lombok.AllArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
 import org.hibernate.SessionFactory;
 import org.hibernate.context.internal.ManagedSessionContext;
 
 @AllArgsConstructor
+@Slf4j
 public abstract class TransactionRunner<T> {
     private MultiTenantUnitOfWorkAwareProxyFactory proxyFactory;
     private SessionFactory sessionFactory;
     private ConstTenantIdentifierResolver tenantIdentifierResolver;
 
-    public T start(boolean reUseSession, UnitOfWork unitOfWork) throws Throwable {
+    public T start(boolean reUseSession, UnitOfWork unitOfWork, String operation) throws Throwable {
```

Comment:
You mean ```context```?

Reply:
`operation` was name taken from similar instrumentation being done on filing app.